ci/travis/build-docker-images.sh

```diff
@@ -71,23 +71,25 @@ if [[ "$TRAVIS" == "true" ]]; then
 
     commit_sha=$(echo "$TRAVIS_COMMIT" | head -c 6)
     cp -r "$ROOT_DIR"/.whl "$ROOT_DIR"/docker/ray/.whl
-    cp "$ROOT_DIR"/python/requirements.txt "$ROOT_DIR"/docker/autoscaler/requirements.txt
-    cp "$ROOT_DIR"/python/requirements_autoscaler.txt "$ROOT_DIR"/docker/autoscaler/requirements_autoscaler.txt
+    cp "$ROOT_DIR"/python/requirements.txt "$ROOT_DIR"/docker/ray-ml/requirements.txt
+    cp "$ROOT_DIR"/python/requirements_ml_docker.txt "$ROOT_DIR"/docker/ray-ml/requirements_ml_docker.txt
+    cp "$ROOT_DIR"/python/requirements_tune.txt "$ROOT_DIR"/docker/ray-ml/requirements_tune.txt
+    cp "$ROOT_DIR"/python/requirements_rllib.txt "$ROOT_DIR"/docker/ray-ml/requirements_rllib.txt
```

Comment:
I think glob syntax should still work? `cp "$ROOT_DIR"/python/requirements*.txt "$ROOT_DIR"/docker/ray-ml/`

Reply:
Yep! That should do it!